spec/controllers/publics_controller_spec.rb

```diff
@@ -42,7 +42,7 @@
       post :receive, "guid" => @user.person.guid.to_s, "xml" => xml
     end
 
-    it 'unescapes the xml before sending it to receive_salmon' do
+    it 'unescapes the xml before sending it to receive' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.